<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>1001</width>
     <height>801</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="roomLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>230</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Wybierz swój pokój:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="roomSubmitButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>600</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Potwierdź</string>
    </property>
   </widget>
   <widget class="QComboBox" name="roomComboBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>280</y>
      <width>441</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Pokój 1 (0/9)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pokój 2 (0/9)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pokój 3 (0/9)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pokój 4 (0/9)</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
